src/frontend/src/modules/Client/Devices.vue

```diff
@@ -64,10 +73,34 @@ export default {
   mounted: function () {
     EventBus.$emit("widgetContentLoaded", this.subscriber, this.devices.length)
   },
-  methods: {
+  /*************  ✨ Windsurf Command ⭐  *************/
+  /**
+   * Emit event to set map center for the given device
+   *
+   * @param {Object} device - device object
+   */
+  /*******  b22be44d-29b2-44df-8ac2-f22996c0ee68  *******/ methods: {
```

Comment:
This looks like a leftover from development. @munyanezaarmel can you remove it 😉 ?